src/options/options.js

```diff
@@ -25,3 +29,18 @@ document.getElementById('configuration').onsubmit = function (event) {
     }, 1500);
   });
 };
+// options/options.js
+
+document.getElementById('save-disable-links').addEventListener('click', () => {
+  const textarea = document.getElementById('disableUrls');
+  const rawLines = textarea.value.split('\n');
+
+  const patterns = rawLines.map((l) => l.trim()).filter((l) => l.length > 0);
```

Comment:
Consider naming your variables with something more readable
```suggestion
  const patterns = rawLines.map((line) => line.trim()).filter((line) => line.length > 0);
```